src/python/WMComponent/WorkflowUpdater/WorkflowUpdaterPoller.py

```diff
@@ -303,7 +304,10 @@ def algorithm(self, parameters=None):
             # and resolve blocks in each container being used by workflows
             # considerations for 2024 are around 100 pileups each taking 2 seconds in Rucio
             with CodeTimer("Rucio block resolution", logger=logging):
-                self.findRucioBlocks(uniqueActivePU, msPileupList)
+                try:
+                    self.findRucioBlocks(uniqueActivePU, msPileupList)
```

Comment:
Given that the actual code in `findRucioBlocks()` has already been encasulated in a try/except block, I don't think this one is needed.

Reply:
ok, done